### DISCUSSION TITLE
[Beta] multipath-ietf-draft-01-beta.1

### DISCUSSION THREAD

**Discussion by ruiqizhou** [alibaba/xquic#231] [Announcements]
Support [draft-ietf-quic-multipath-01](https://datatracker.ietf.org/doc/html/draft-ietf-quic-multipath-01)

```
Attention:
* don't use xqc_bbr_cb or xqc_bbr2_cb
* default scheduler: minRTT
* the current reinjection mode is very aggressive, please use with caution.
```

<hr /><em>This discussion was created from the release <a href='https://github.com/alibaba/xquic/releases/tag/multipath-draft-01-beta.1'>[Beta] multipath-ietf-draft-01-beta.1</a>.</em>

**Comment by jeff277** [comment]
是否会考虑类似于 MPTCP的冗余模式？

v0.92新增功能, 原生就支持 路径管理选择FullMesh，调度模式选择 redundant。

 http://multipath-tcp.org/pmwiki.php/Users/ConfigureMPTCP 'redundant': This scheduler will try to transmit the traffic on all available subflows in a redundant way. It is useful when one wants to achieve the lowest possible latency by sacrificing the bandwidth.

**Reply by ruiqizhou** [reply]
后续版本会考虑支持该特性

**Reply by jeff277** [reply]
请问 类似于 MPTCP的冗余模式，大约什么时候开始支持？
（有一个项目，一直想用）
